services/src/main/java/org/jboss/windup/web/services/model/MigrationProject.java

```diff
@@ -67,7 +67,7 @@
 
     @Column(length = 120, unique = false, nullable = false)
     @Size(min = 3, max = 120, message = "The name must be at least 3 letters long.")
-    @Pattern(message = "The name must contain only alphanumeric characters",
+    @Pattern(message = "The name must contain only alphanumeric characters and spaces",
             regexp = "\\s*\\w[- \\w]+\\w\\s*") // Allow whitespace around, limit to A-Za-z0-9-, allow spaces inside, at least 3 characters after whitespace trimmed.
```

Comment:
I don't believe that this was in your original PR. That is actually one of the things that were commented on that you didn't change.